Packs/MicrosoftExchangeOnline/ReleaseNotes/1_2_38.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### EWS O365
+
+- Fixed an issue where the fetch command have crashed when handling email attachments with an invalid header format.
```

Comment:
```suggestion
- Fixed an issue where fetching failed when email attachments had headers with an invalid format (`<[invalid_value]>` instead of `<valid_value>`), by removing the square brackets.
```

Reply:
Fixed.